config/config_env.go

```diff
@@ -52,7 +52,10 @@ const (
 // FromEnv uses environment variables to set the tracer's Configuration
 func FromEnv() (*Configuration, error) {
 	c := &Configuration{}
+	return c.FromEnv()
+}
 
+func (c *Configuration) FromEnv() (*Configuration, error) {
```

Comment:
Lint Failures
config/config_env.go:58:1: exported method Configuration.FromEnv should have comment or be unexported

Reply:
run `make lint`